### ISSUE TITLE
[Suggestion] Hide "Topics to follow"

### ISSUE THREAD

**Issue by sonicspike41** [oslego/better-twitter#17]
![image](https://user-images.githubusercontent.com/24579737/91130865-f5567b80-e671-11ea-836e-819b281af6fb.png)
URL: https://twitter.com/drmistercody/with_replies
Browser: Firefox 79.0 (64-bit)
Windows 10 Home (Version 1909 OS Build 18363.1016)
BT Version: 2.0.1

HTML of "Topics to follow" header section:
![image](https://user-images.githubusercontent.com/24579737/91135345-9d207900-e673-11ea-8f31-9a680737c711.png)

HTML of individual topic:
![image](https://user-images.githubusercontent.com/24579737/91136246-e1137e00-e673-11ea-9b02-dd829cf7ebfc.png)
And follow button since I ran out of room above:
![image](https://user-images.githubusercontent.com/24579737/91136596-fa1c2f00-e673-11ea-9af4-666cdcd45aaf.png)

HTML of "More Topics" link:
![image](https://user-images.githubusercontent.com/24579737/91137053-1d46de80-e674-11ea-8263-37450a78214c.png)

**Comment by oslego** [comment]
Sounds like a good idea. I'm bothered by the topics list too.

**Comment by privacyguy123** [comment]
+1 for this

**Comment by azat11** [comment]
Yeah, that's the only thing crewing up the feed. Please add that.

Grateful for the extension, wouldn't be able to use Twitter without it.

**Comment by Reggie521** [comment]
Any update on adding this option?

**Comment by oslego** [comment]
@Reggie521 No updates. I am not able to work on the extension at this time.
